ProjectSidewalk/SidewalkWebpage
# 2584 shorten labels uri

Resolves #2584

Shortened the uri by converting the loadedLabels parameter to a body parameter. I wrote some "hacky" code to get around the `Redirect("anonSignUp?url=...")` as this function only ran GET requests. To get around this, I created a fake GET endpoint that auto-passes and reran the request manually.

##### Before/After screenshots (if applicable)

##### Testing instructions
I am not sure how to test this as I do not have more than a few pages of labelled images on localhost. However, to test that my code does not affect the gallery:
1. Go to `/gallery`
2. Make sure images still load
3. Clear cookies and reload to make sure anonomous signup still works
